snap/local/start-mysqld-exporter.sh

```diff
@@ -18,12 +18,13 @@ EXPORTER_PATH="/usr/bin/prometheus-mysqld-exporter"
 SOCKET="/var/run/mysqld/mysqld.sock"
 
 if [ -z "$SNAP" ]; then
-    # When not running as a snap, expect `DATA_SOURCE_NAME` to be set.
-    if [ -z "$DATA_SOURCE_NAME" ]; then
-        echo "DATA_SOURCE_NAME must be set"
+    # When not running as a snap, expect `EXPORTER_USER` and `EXPORTER_PASS` to be set.
+    if [ -z "$EXPORTER_USER" || -z "$EXPORTER_PASS" ]; then
+        echo "EXPORTER_USER and EXPORTER_PASS must be set"
         exit 1
     fi
-    exec "$EXPORTER_PATH" "${EXPORTER_OPTS[@]}"
+    exec env MYSQLD_EXPORTER_PASS="${EXPORTER_PASS}" \
+        "$EXPORTER_PATH" "--mysqld.username=${EXPORTER_USER}" "--mysqld.address=unix://${SOCKET}" "${EXPORTER_OPTS[@]}"
```

Comment:
Just layout for better readability:
```suggestion
    exec env MYSQLD_EXPORTER_PASS="${EXPORTER_PASS}" \
        "${EXPORTER_PATH}" \
        "--mysqld.username=${EXPORTER_USER}" \
        "--mysqld.address=unix://${SOCKET}" \
        "${EXPORTER_OPTS[@]}"
```

Reply:
addressed in fa4e9a8c4b064b02274bd4bfa528966a8e690469